packages/grafana-ui/src/components/PanelChrome/PanelChrome.tsx

```diff
@@ -120,56 +148,38 @@ export function PanelChrome({
         ) : null}
       </div>
 
-      <div className={cx(styles.headerContainer, dragClass)} style={headerStyles} data-testid="header-container">
-        {title && (
-          <h6 title={title} className={styles.title}>
-            {title}
-          </h6>
-        )}
-
-        <PanelDescription description={description} className={dragClassCancel} />
-
-        {titleItems.length > 0 && (
-          <div className={cx(styles.titleItems, dragClassCancel)} data-testid="title-items-container">
-            {titleItems.map((item) => item)}
-          </div>
-        )}
-
-        {loadingState === LoadingState.Streaming && (
-          <Tooltip content="Streaming">
-            <TitleItem className={dragClassCancel} data-testid="panel-streaming">
-              <Icon name="circle-mono" size="md" className={styles.streaming} />
-            </TitleItem>
-          </Tooltip>
-        )}
-
-        <div className={styles.rightAligned}>
-          {menu && (
-            <Dropdown overlay={menu} placement="bottom">
-              <ToolbarButton
-                aria-label={`Menu for panel with ${title ? `title ${title}` : 'no title'}`}
-                title="Menu"
-                icon="ellipsis-v"
-                iconSize="md"
-                narrow
-                data-testid="panel-menu-button"
-                className={cx(styles.menuItem, dragClassCancel, 'menu-icon')}
+      {(hoverHeader || !hasHeader) && menu && (
+        <HoverWidget menu={menu} title={title} offset={hoverHeaderOffset} dragClass={dragClass}>
```

Comment:
Have we tried to put the hover widget on a Popover and use the dashboard as a boundary? I know it is quite complicated, so this solution is much simpler and makes it work for our case.

However, if we use the PanelChrome out of Dashboards, we will have the same problem, while if we use Poppover, we will solve it for any place where the PanelChrome is placed. I don't think this is the PR to implement this, but I'm curious if we have tried.